demos/paint_your_dreams_demo/main.py

```diff
@@ -39,17 +41,21 @@ def get_available_devices() -> list[str]:
     return list({device.split(".")[0] for device in core.available_devices if device != "NPU"})
 
 
-def download_models(model_name: str, safety_checker_model: str) -> None:
+def download_models(model_name_t2i: str, model_name_i2i: str, safety_checker_model: str) -> None:
     global safety_checker
 
-    is_openvino_model = model_name.split("/")[0] == "OpenVINO"
+    is_openvino_model = model_name_t2i.split("/")[0] == "OpenVINO"
 
-    output_dir = MODEL_DIR / model_name
-    if not output_dir.exists():
+    output_dir_t2i = MODEL_DIR / model_name_t2i
+    if not output_dir_t2i.exists():
         if is_openvino_model:
-            snapshot_download(model_name, local_dir=output_dir)
+            snapshot_download(model_name_t2i, local_dir=output_dir_t2i, resume_download=True)
         else:
-            raise ValueError(f"Model {model_name} is not from OpenVINO Hub and not supported")
+            raise ValueError(f"Model {model_name_t2i} is not from OpenVINO Hub and not supported")
+        
+    output_dir_t2i = MODEL_DIR / model_name_i2i
+    if not output_dir_t2i.exists():
+        os.system("optimum-cli export openvino --model " + str (model_name_i2i) + " --task stable-diffusion --weight-format fp16 " + str(output_dir_t2i))
```

Comment:
I'm not a big fan of combining bash and python. Could we achieve the same with python code only?